criu/pipes.c

```diff
@@ -282,7 +282,7 @@ static char *pipe_d_name(struct file_desc *d, char *buf, size_t s)
 	struct pipe_info *pi;
 
 	pi = container_of(d, struct pipe_info, d);
-	if (snprintf(buf, s, "pipe:[%d]", pi->pe->pipe_id) >= s) {
+	if (snprintf(buf, s, "pipe:[%u]", pi->pe->pipe_id) >= s) {
```

Comment:
perhaps we should also change the print format in the error message on the next line?

Reply:
Fixed. Thanks!